fab/utils/prioritised_replay_buffer.py

```diff
@@ -95,6 +95,9 @@ def sample_n_batches(self, batch_size: int, n_batches: int) -> \
     def adjust(self, log_w_adjustment, log_q, indices):
         """Adjust log weights and log q to match new value of theta, this is typically performed
         over minibatches, rather than over the whole dataset at once."""
+        valid_adjustment = ~torch.isinf(log_w_adjustment) & ~torch.isnan(log_q)
+        log_w_adjustment, log_q, indices = \
+            log_w_adjustment[valid_adjustment], log_q[valid_adjustment], indices[valid_adjustment]
```

Comment:
In the GMM, because the flow has very poor initialisation, I sometimes get NaN's here. So I have added this to improve stability.